### PR TITLE
Update scripts to use the broker apb

### PR DESCRIPTION
**Pull request by djzager**
Update scripts that deploy the broker to use the [`automation-broker-apb`](https://github.com/automationbroker/automation-broker-apb).

### CODE REVIEW

File: scripts/run_latest_k8s_build.sh [hunk 1]
```diff
@@ -1,58 +1,31 @@
...
-
-    context=$(kubectl config current-context)
-    cluster=$(kubectl config get-contexts $context --no-headers | awk '{ print $3 }')
-
-    kubectl config set-context $context --cluster=$cluster --namespace=ansible-service-broker
-    kubectl create -f "/tmp/k8s-ansible-service-broker.yaml"
-
-    create-broker-resource
+    kubectl create ns $BROKER_NAMESPACE
+    kubectl create serviceaccount $APB_NAME --namespace $BROKER_NAMESPACE
```
Comment (shawn-hurley) [inline scripts/run_latest_k8s_build.sh]: Can we delete this svcacct after this runs? 

Also could this just run as the user running the script? do we want to do this extra work?

File: scripts/run_latest_build.sh [hunk 2]
```diff
@@ -41,65 +41,66 @@ HOSTNAME=${PUBLIC_IP}.nip.io
...
-
-curl -s $TEMPLATE_URL \
-  | oc process \
-  -n ansible-service-broker \
-  -p DOCKERHUB_ORG="$DOCKERHUB_ORG" \
-  -p ENABLE_BASIC_AUTH="$ENABLE_BASIC_AUTH" \
-  -p NAMESPACE=ansible-service-broker \
-  $VARS -f - | oc create -f -
+# Use the automation-broker-apb to deploy the broker
+oc create serviceaccount $APB_NAME --namespace $BROKER_NAMESPACE
```
Comment (shawn-hurley) [inline scripts/run_latest_build.sh]: Same question as below re: do we need the service account

Comment (djzager) [reply scripts/run_latest_build.sh]: I actually was smart and delete that stuff in this script but forgot to add that to the k8s version.

Ref: https://github.com/openshift/ansible-service-broker/pull/891/files#diff-34011cced263e7ac1afc542ee01bf933R101

File: scripts/run_latest_build.sh [hunk 3]
```diff
@@ -41,65 +41,65 @@ HOSTNAME=${PUBLIC_IP}.nip.io
...
+    --serviceaccount=$APB_NAME \
+    -- provision -e broker_name=$BROKER_NAME
 if [ "$?" -ne 0 ]; then
-  echo "Error processing template and creating deployment"
+  echo "Error deploying broker"
   exit
 fi
+oc delete pod -n $BROKER_NAMESPACE $APB_NAME
+oc delete clusterrolebinding $APB_NAME
+oc delete serviceaccount $APB_NAME
```
Comment (shawn-hurley) [inline scripts/run_latest_build.sh]: I think your going to want to specify the namespace for this deletion as well.

**Pull request merged**